staging/publishing/rules.yaml

```diff
@@ -26,6 +26,12 @@ rules:
       dir: staging/src/k8s.io/code-generator
     name: release-1.17
     go: 1.13.4
+  - source:
+      branch: release-1.18
+      dir: staging/src/k8s.io/code-generator
+    name: release-1.18
+    go: 1.13.4
```

Comment:
@liggitt -- Should these be 1.13.6?

Reply:
it doesn't really matter, this is just for the bot to test with, and 1.13.x is sufficient. If we want to update this we should do it separately and update the levels for all the branches

Reply:
SGTM. We can take care of that after the 1.13.7 bump.